dashboard/lib/pd/workshop_survey_foorm_constants.rb

```diff
@@ -0,0 +1,25 @@
+module Pd
+  module WorkshopSurveyFoormConstants
+    include SharedWorkshopConstants
+
+    DAILY_SURVEY_NAMES = {
```

Comment:
Is "names" the right word for this? Like `DAILY_SURVEY_CONFIG_PATHS` or something?

Reply:
good call